app/src/main/java/org/sopt/seminar/util/MyTouchHelperCallback.kt

```diff
@@ -0,0 +1,57 @@
+package org.sopt.seminar.util
+
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.seminar.presentation.follower.FollowerAdapter
+
+class MyTouchHelperCallback(
+    private val itemMoveListener: OnItemMoveListener
+) : ItemTouchHelper.Callback() {
+
+    private var isMoved = false
+
+    interface OnItemMoveListener {
+        fun onItemMove(fromPosition: Int, toPosition: Int)
+        fun startDrag(onStartDragListener: FollowerAdapter.OnStartDragListener) {}
+        fun onItemSwipe(position: Int)
+    }
+
+    /**
+     * 어느 방향으로 움직일지에 따라서 flag 받는것을 정의
+     * 드래그는 위, 아래 액션이기 때문에 up, down 을 넘겨줌
+     */
+    override fun getMovementFlags(
+        recyclerView: RecyclerView,
+        viewHolder: RecyclerView.ViewHolder
+    ): Int {
+        val dragFlags = ItemTouchHelper.UP or ItemTouchHelper.DOWN
+        return makeMovementFlags(dragFlags, 0)
+    }
+
+    /**
+     * 어느 위치에서 어느 위치로 변경하는지
+     */
+    override fun onMove(
+        recyclerView: RecyclerView,
+        viewHolder: RecyclerView.ViewHolder,
+        target: RecyclerView.ViewHolder
+    ): Boolean {
+        itemMoveListener.onItemMove(viewHolder.adapterPosition, target.adapterPosition)
+        isMoved = true
+        return true
+    }
+
+    override fun onSelectedChanged(viewHolder: RecyclerView.ViewHolder?, actionState: Int) {
+        super.onSelectedChanged(viewHolder, actionState)
+        if (isMoved) {
+            isMoved = false
+            val followAdapter = FollowerAdapter()
+            followAdapter.afterDragAndDrop()
```

Comment:
`afterDragAndDrop()` 메서드 내부는 `nofityDataSetChanged()` 한 줄만 있으니, 현우님 리뷰처럼 private으로 돌리거나 저라면 그냥 이렇게 할 것 같습니다.
```suggestion
            followAdapter.notifyDataSetChanged()
```